restclient/sender.go

```diff
@@ -1,6 +1,10 @@
 package restclient
 
+// Sender is an interface used to send a message to the guble server.
 type Sender interface {
-	Send(topic string, body []byte) error
+	// Send a a message(body) to the guble Server, to the given topic, with the given userID.
+	Send(topic string, body []byte, userID string) error
```

Comment:
Shouldn't user be added at construction time? It shouldnt change across the life cycle of a sender, right?